INSTALL.md

```diff
@@ -14,4 +14,75 @@ You can run tests with:
 cargo test
 ```
 
-Please refer to the cargo documentation for more detailed instructions.
\ No newline at end of file
+Please refer to the cargo documentation for more detailed instructions.
+
+
+### If you want to create a system service to run your application follow the instructions below:
```

Comment:
I'd move this inside the `Systemd setup for backend` section.

Also, If you want -> Should you want to / If you would like to

Reply:
Done